internal/web_api.go

```diff
@@ -16,13 +16,16 @@ import (
 
 const apiVersion = "3.2"
 
+type RetryStrategyFunc func(uint, error) bool
```

Comment:
pls add docs for this, especially what uint is and if its enumerated from 0 or 1 :)

Reply:
:pepe_grand: